hoomd/mpcd/tune.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2009-2023 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+r"""MPCD tuning operations.
+
+These operations will affect the performance of MPCD simulations but not
+their correctness.
+
+"""
+
+import hoomd
+from hoomd.mpcd import _mpcd
+from hoomd.operation import Tuner
+
+
+class ParticleSorter(Tuner):
+    r"""MPCD particle sorter.
+
+    Args:
+        trigger (hoomd.trigger.trigger_like): Number of integration steps
+            between sorting.
+
+    This tuner sorts the MPCD particles into cell order. To perform the sort,
+    the cell list is first computed with the current particle order. Particles
+    are then reordered in memory as they are stored in the cell list, which can
+    significantly improve performance of all subsequent cell-based steps of the
+    MPCD algorithm due to improved cache coherency.
+
+    The optimal frequency for sorting depends on the number of particles, so the
+    `trigger` itself should be tuned to give the maximum performance. The
+    `trigger` should be a multiple of `hoomd.mpcd.collide.CollisionMethod.period`
+    to avoid unnecessary cell list builds. Typically, using a small multiple
+    (tens) of the collision period works best.
```

Comment:
```suggestion
    trigger's period should be a multiple of `hoomd.mpcd.collide.CollisionMethod.period`
    to avoid unnecessary cell list builds. Typically, using a small multiple
    (tens) of the collision period works best.
```